### PR TITLE
added printconfirm

### PR DESCRIPTION
**Pull request by flighter18**
did a new git clone (version 1.8.1) and added the print confirmation. 
should now work.

### CODE REVIEW

**Comment by andi34** [comment]
Thx :) I'll check next days. Quite busy because my GF and her daughter going to move to my apartment soon

**Comment by flighter18** [comment]
perfect thank you! And good luck :)

File: lang/fr.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ L10N = {
 	"error": "Il y a eu un problème. Merci de réessayer",
 	"reload": "Recharger la page",
 	"print": "Imprimer",
+	"printing": "Imprimer commencé",
```
Comment (thymon13) [inline lang/fr.js:23]: "printing": "L'impression a commencé",

**Review by thymon13 — CHANGES_REQUESTED** [review]
correction : French translation.

**Comment by andi34** [comment]
Please keep this open until I've done a decision which way to go (swal or no swal)

**Comment by andi34** [comment]
Different solution added.
Thanks for this solution too, maybe we can use it later for other things. 🙂

**Pull request closed without merging**